SpatialGDK/Documentation/content/non-unreal-layers.md

```diff
@@ -129,48 +128,31 @@ Worker_RequestId SendSomeCommandRequest(Worker_EntityId TargetEntityId, Worker_C
 #### Receive data
 You could receive and deserialize a component update and command request in your Unreal project code in the following way:
 ```
-UCLASS()
-class SPATIALGDK_API SessionComponentCallbacks : UOpCallbackTemplate
+void ATPSGameMode::BeginPlay()
```

Comment:
Slightly concerned about the race here between BeginPlay being called, and the NetDriver being setup. I believe if you test this code a switch off dedicated server it'll crash as there shouldn't be a net driver yet.